cmd/kubeadm/app/cmd/phases/workflow/runner.go

```diff
@@ -295,10 +295,6 @@ func (e *Runner) SetAdditionalFlags(fn func(*pflag.FlagSet)) {
 // command help, adding phase related flags and by adding phases subcommands
 // Please note that this command needs to be done once all the phases are added to the Runner.
 func (e *Runner) BindToCommand(cmd *cobra.Command) {
-	if len(e.Phases) == 0 {
-		return
-	}
-
```

Comment:
@fabriziopandini i had to do this to unblock the PR in terms of not having any phases yet.
otherwise with this check `BindToCommand()` returned early and the setup was incomplete, causing a panic in a `cmd.Flags()` call.

i think this removal is fine, runner wise.

Reply:
outdated.